### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by aakarshitpandey**
Added the resources page. Currently has the sample resources taken from the google drive folder.

### CODE REVIEW

File: frontend/modules/resources/resources.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  {
+    name: "Purdue Hackers",
+    workshop: [
+      {
+        title: "Nodejs Workshop",
+        link:
+          "https://purduehackers.gitbook.io/tutorials/workshops/nodejs-workshop"
+      },
+    ],
+    image: "../../static/images/nodejslogo.png",
```
Comment (ashwinGokhale) [inline frontend/modules/resources/resources.tsx]: Remove the "../../" in the file path for all of these file paths

File: package.json [hunk 2]
```diff
@@ -99,6 +99,7 @@
 		"enzyme": "^3.8.0",
 		"enzyme-adapter-react-16": "^1.8.0",
 		"faker": "^4.1.0",
+		"file-loader": "^3.0.1",
```
[the remark this answers is not in the document]
Comment (aakarshitpandey) [reply package.json]: Done!

File: frontend/modules/resources/resources.tsx (hunk 1 above)
Comment (aakarshitpandey) [reply frontend/modules/resources/resources.tsx]: Done!

**Comment by aakarshitpandey** [comment]
Made the required changes to the resource page

**Pull request merged**